common/environment/environment.cpp

```diff
@@ -1821,9 +1791,45 @@ CConstDropZoneServerInfoIterator::CConstDropZoneServerInfoIterator(const IConstD
 {
     Owned<IEnvironmentFactory> factory = getEnvironmentFactory();
     constEnv.setown((CLocalEnvironment *)factory->openEnvironment());
-    Owned<IPropertyTree> pSrc = &dropZone->getPTree();
-    serverListIt.setown(pSrc->getElements("ServerList"));
-    maxIndex = pSrc->getCount("ServerList");
+
+    // For backward compatibility
+    SCMStringBuffer dropZoneMachineName;
+    // Returns dropzone '@computer' value if it is exists
+    dropZone->getComputerName(dropZoneMachineName);
+
+    if (0 != dropZoneMachineName.length())
+    {
+        // Create a ServerList for legacy element.
+        legacyServerList.setown(createPTree());
+
+        Owned<IConstMachineInfo> machineInfo = constEnv->getMachine(dropZoneMachineName.str());
+        if (machineInfo)
+        {
+            SCMStringBuffer dropZoneMachineNetAddress;
+            machineInfo->getNetAddress(dropZoneMachineNetAddress);
+
+            // Create a single ServerList record related to @computer
+            //<ServerList name="ServerList" server="<IP_of_@computer>"/>
+            Owned<IPropertyTree> newRecord = createPTree();
+            newRecord->setProp("@name", "ServerList");
```

Comment:
@AttilaVamos - I've checked with @rpastrana , it doesn't matter what it is,  configmgr / xslt generation needed it to be a unique something, but here for your internally generated version it can be anything.